storage/interfaces.go

```diff
@@ -37,6 +37,7 @@ type Settings struct {
 	URI           string
 	MigrationsURL string `mapstructure:"migrations_url"`
 	EncryptionKey string `mapstructure:"encryption_key"`
+	SkipSSL       bool   `mapstructure:"skip_ssl"`
```

Comment:
how about naming it skip_ssl_validation so that we are consistent with the api skip ssl validation?